### PR TITLE
UI Tests and Refactor 2

### PR DESCRIPTION
**Pull request by alisnichenko**
### What's new
* Added UI tests: testers can refer to the `test_ui.py` file for an example.
* Refactored small portions of the code, more refactoring is needed. Although it is cleaner now. Specifically added new lines where needed, limited the code to 80 characters, changed camel case variables to snake case to conform to the rest of the content, moved print statements to utils file, etc.
* Overall changed the testing infrastructure to allow for UI testing.

### Issues
#27 is addressed partially in this PR, since UI tests haven't been implemented.

### CODE REVIEW

File: test/test_ui.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import in_college
+
+# Create actual instance of config object.
+config = utils.InCollegeConfig(config_path)
+
+def test_connect_friends(monkeypatch, capsys):
+    """Example UI test with captured output and simulated keypresses."""
+    test_sequence = [
+            key.ENTER,      # go to connect friends. 
+            *list('admin'), # enter friend's first name.
```
Comment (Peacewalker365) [inline test/test_ui.py:36]: Can I ask why you list() it and then unpack it? I know you must have some reason for this. But I don't understand what's the diff between 'admin' and *list('admin').

**Review by Peacewalker365 — APPROVED** [review]
Great work man!!!

File: test/test_ui.py (hunk 1 above)
Comment (alisnichenko) [reply test/test_ui.py:36]: Each item in the `test_sequence` array is a key press. As such, strings, such as logins and passwords must be represented as lists: `'a', 'd', 'm', 'i', 'n'`. By creating a list from a string and unpacking it we are putting each character in the same list separately. I could just put all letters separately, maybe I will do it later, but that was concise enough for now. You can change it thought :eyes: I am sure some issues will pop up when more tests are created.

Comment (alisnichenko) [reply test/test_ui.py:36]: Oh, comment about the difference:
```python
# Putting in a list
['admin']
>>> ['admin']
[*list('admin')]
>>> ['a', 'd', 'm', 'i', 'n']

Comment (Peacewalker365) [reply test/test_ui.py:36]: Cool, I got the idea. Thx!

**Pull request merged**